reference/conan_server.rst

```diff
@@ -86,7 +86,7 @@ Server Parameters
   
     ..  code-block:: bash 
         
-        docker run ... -p9300:9999 ... # Check Docker docs for that
+        docker run ... -p9999:9300 ... # Check Docker docs for that
```

Comment:
But the ``conan_server`` is running on port 9300 by default, so the inside port should be 9300, isn't it? What am I missing?

Reply:
Left is "outside" right is "inside" :)

Reply:
that was it, the text was confusing, thanks for clarifying!